### PR TITLE
[feature][connector-v2-hbase-sink] Support Connector v2 HBase sink TTL data writing

### PR DESCRIPTION
**Pull request by zhangshenghang**
### Purpose of this pull request

In some scenarios, you need to set the TTL time

### How was this patch tested?

Local and server deployment operation

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java [hunk 1]
```diff
@@ -97,6 +97,9 @@ public class HbaseConfig {
                     .noDefaultValue()
                     .withDescription("Hbase extra config");
 
+    public static final Option<Long> HBASE_TTL_CONFIG =
+            Options.key("ttl").longType().defaultValue(-1L).withDescription("Hbase ttl config");
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java]:
```suggestion
            Options.key("ttl").longType().defaultValue(-1L).withDescription("The expiration time configuration for writing hbase data. The default value is -1, indicating no expiration time.");
```

**Review by Hisoka-X — COMMENTED** [review]
Please add this new config key into doc

File: seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java (hunk 1 above)
Comment (zhangshenghang) [reply seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java]: > 

Thank you for your guidance. The modifications have been completed

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java]: https://github.com/apache/seatunnel/pull/7116#pullrequestreview-2161342401

Comment (zhangshenghang) [reply seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java]: > [#7116 (review)](https://github.com/apache/seatunnel/pull/7116#pullrequestreview-2161342401)

Add Complete

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**